src/parser/jobs/whm/modules/oGCDs.tsx

```diff
@@ -139,9 +145,10 @@ export default class OGCDs extends Module {
 		}))
 
 		OTHER_COOLDOWNS_TRACKED.forEach(action => {
+			const cooldownUsage = this.getCooldownUsage(action.id)
 			const maxUses = Math.ceil(this.parser.fightDuration / (action.cooldown * 1000))
-			const uses = this.cooldownUsage[action.id].uses
-			const held = uses > 0 ? this.cooldownUsage[action.id].held : pullDuration
+			const uses = cooldownUsage.uses
+			const held = uses > 0 ? cooldownUsage.held : pullDuration
```

Comment:
I'm still not sure about this, as I wrote in Discord, it doesn't get closed out for end of fight. I think if you took each window and did a final pass at the start of `onComplete()`, for anything that came off CD earlier than end of fight minus the CD, and set held based on that, you'd catch any missed final windows, and the held time would be pullDuration if they never used it anyway.